build.gradle

```diff
@@ -754,6 +754,8 @@ subprojects {
                     '-Xep:EqualsGetClass:OFF',
                     // Not a real problem
                     '-Xep:MixedMutabilityReturnType:OFF',
+                    // Don't want to add a dependency to ErrorProne.
+                    '-Xep:AnnotateFormatMethod:OFF',
```

Comment:
Instead, we should enable to Format String Checker https://checkerframework.org/manual/#formatter-checker
This would dog-food another checker and give use the same guarantees.
Maybe this is a nice simple starter task?

Reply:
Independently, there is https://github.com/google/error-prone/issues/1440 .

Reply:
I'll try adding Checker Framework FormatMethod annotations where ErrorProne suggest and then I'll try running the Format Checker.